### PR TITLE
[chore] delete skipped test

### PR DESCRIPTION
**Pull request by atoulme**
This test has been skipped for a bit, prefer to delete it at this time.

### CODE REVIEW

File: test/k8s_logging_tests/test_config_logging.py [hunk 1]
```diff
@@ -108,34 +108,6 @@ def test_annotation_excluding(setup, container_name, expected):
                 len(events))
     assert len(events) == expected
 
-@pytest.mark.skipif(True, reason="Jira: ADDON-36296")
```
Comment (rmfitzpatrick) [inline test/k8s_logging_tests/test_config_logging.py:111]: this ticket is closed. do we know the tests still need to be skipped?

Comment (atoulme) [reply test/k8s_logging_tests/test_config_logging.py:111]: it was closed for inactivity.

**Comment by atoulme** [comment]
@omrozowicz-splunk please review when you get a chance

**Review by dmitryax — APPROVED** [review]

**Comment by atoulme** [comment]
Merging - @omrozowicz-splunk we can discuss offline if this needs a follow-up.

**Pull request merged**